generate_items/wowhead_api.go

```diff
@@ -524,7 +524,7 @@ var gemSocketColorPatterns = map[proto.GemColor]*regexp.Regexp{
 	proto.GemColor_GemColorOrange:    regexp.MustCompile("Matches a ((Yellow)|(Red)) or ((Yellow)|(Red)) (S|s)ocket\\."),
 	proto.GemColor_GemColorPurple:    regexp.MustCompile("Matches a ((Blue)|(Red)) or ((Blue)|(Red)) (S|s)ocket\\."),
 	proto.GemColor_GemColorGreen:     regexp.MustCompile("Matches a ((Yellow)|(Blue)) or ((Yellow)|(Blue)) (S|s)ocket\\."),
-	proto.GemColor_GemColorPrismatic: regexp.MustCompile("Matches a Red, Yellow or Blue (S|s)ocket\\."),
+	proto.GemColor_GemColorPrismatic: regexp.MustCompile("Matches a Red, Yellow or Blue (S|s)ocket\\."|"Matches any socket\\."|"Matches any Socket\\.  Maximum of one socketed in your equipment"),
```

Comment:
get rid of the " marks in there. Should just be around the string itself

Reply:
and probably dont need `Matches any Socket\\.  Maximum of one socketed in your equipment` at the end since the first one `Matches any socket\\.` would have already caught it.